app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeViewModel.kt

```diff
@@ -115,13 +116,13 @@ class AttendeeViewModel(private val attendeeService: AttendeeService, private va
                 .subscribeOn(Schedulers.io())
                 .observeOn(AndroidSchedulers.mainThread())
                 .subscribe({
-                    tickets.value = it
+                    tickets.value = it as ArrayList<Ticket>?
                 }, {
                     Timber.e(it, "Error Loading tickets!")
                 }))
     }
 
-    fun createAttendee(attendee: Attendee, eventId: Long, country: String, paymentOption: String) {
+    fun createAttendee(attendee: Attendee, eventId: Long, country: String, paymentOption: String, totalAttendee: Int) {
```

Comment:
It's strange every method has so many parameters as if all VM methods are static

However most data should already be in VM not requiring any parameters from view except maybe 2-3